news/11256.process.rst

```diff
@@ -0,0 +1,2 @@
+Use ``installer`` instead of ``setuptools`` internals to seed virtual envitonments in
```

Comment:
```suggestion
Use ``installer`` instead of ``setuptools`` internals to seed virtual environments in
```

Reply:
Does this need to be called out in our changelog, at the top of everything else?

Reply:
Maybe not. We can skip news for this one and call it out later when *our test suite is compatible with the latest setuptools*.

Reply:
Removed news file.